3.0-HISTORY.rst

```diff
@@ -0,0 +1,11 @@
+3.0.0 (2015-xx-xx)
+++++++++++++++++++
+
+- Remove the ``__bool__`` and ``__nonzero__`` methods from a ``Response``
+  object.
+
+  This has been a planned feature for over a year. The behaviour is exciting
```

Comment:
That's what I get for writing this late at night while talking to a friend about their exciting news.